src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -135,7 +140,11 @@ const Body = BaseForm.extend(Object.assign(
             Logger.error(`Authenticator is not listening on port ${currentPort}.`);
             if (countFailedPorts === ports.length) {
               Logger.error('No available ports. Loopback server failed and polling is cancelled.');
-              this.options.appState.trigger('invokeAction', 'authenticatorChallenge-cancel');
+              this.options.appState.trigger(
+                'invokeAction',
+                this.currentRemediationName === RemediationForms.USER_VERIFICATION_CHALLENGE_POLL ?
+                  'currentAuthenticatorEnrollment-cancel' : 'authenticatorChallenge-cancel'
```

Comment:
currentAuthenticatorEnrollment-cancel ? where does this take the user to ?

Reply:
It will be handled by https://github.com/okta/okta-signin-widget/blob/master/src/v2/controllers/FormController.js#L59 which executes the cancel function in this case.